cuda/test/base/cuda_executor.cu

```diff
@@ -95,6 +98,8 @@ protected:
         cuda = gko::CudaExecutor::create(0, omp);
         cuda2 = gko::CudaExecutor::create(
             gko::CudaExecutor::get_num_devices() - 1, omp);
+        cuda3 = gko::CudaExecutor::create(0, omp, false,
+                                          gko::allocation_mode::unified_host);
```

Comment:
should it also test `unified_global` ?
`unified_host` require device with `cudaDevAttrConcurrentManagedAccess`, or, the device can not access the `unified_host` memory.

Reply:
Ah, yes, sorry. It was meant to be `unified_global` .